.travis.yml

```diff
@@ -11,19 +11,20 @@ before_install:
   - sudo mv docker-compose /usr/local/bin
 before_script:
   - if [[ -f ${CACHE_FILE} ]]; then
-          docker load -i ${CACHE_FILE};
-      else
-          mkdir -p ${CACHE_FOLDER};
-          docker-compose build;
-          docker save web | gzip -c > ${CACHE_FILE};
+        docker load -i ${CACHE_FILE};
+    else
+        mkdir -p ${CACHE_FOLDER};
+        docker-compose build;
+        docker save web | gzip -c > ${CACHE_FILE};
     fi
 script:
   - docker-compose up -d
   - docker-compose exec web /bin/bash -c "composer install && vendor/bin/phpcsoxid --colors"
-  - docker-compose exec web /bin/bash -c "chmod +x ./build/files/test.sh && chmod +x ./build/files/build.sh"
-  - docker-compose exec web /bin/bash -c "./build/files/build.sh 5 $OXID_5_3_RELEASE && ./build/files/test.sh"
-  - docker-compose exec web /bin/bash -c "./build/files/build.sh 6.0 $OXID_6_0_RELEASE && ./build/files/test.sh"
-  - docker-compose exec web /bin/bash -c "./build/files/build.sh 6.1 $OXID_6_1_RELEASE && ./build/files/test.sh"
+  - docker-compose exec web /bin/bash -c "chmod +x ./scripts/build.sh"
```

Comment:
Why did you do change the path?

Reply:
To place all scripts within one directory.